RadixWallet/Core/DesignSystem/Components/Hint.swift

```diff
@@ -73,9 +82,7 @@ public struct Hint: View, Equatable {
 		switch viewState.kind {
 		case .info:
 			nil
-		case .error:
-			AssetResource.error
-		case .warning:
+		case .error, .warning:
```

Comment:
`AssetResource.error` is 17x15 (not squared), which doesn't look great on Hints. Given we are always rendering the image as template and colorizing it, makes more sense to use `AssetResource.warningError` (which is squared).

Reply:
In that case, maybe we can update the `error` asset to be a `24x24` (instead of the current `17x15`)? It is already a template image so it wouldn't break on other places

Reply:
sounds good

Reply:
I just pushed the following changes:
- `error` image was updated to be a square one
- `warning_error` was renamed to `error_large`. The asset is the same than `error`, but it is 4.3x times its size, so it makes sense to have the same name and just the `large` suffix.
- Updated usages of `error_large` (that were actually resizing the image to have `.frame(.smallest)`) to just use `.error` (whose size matches `.smallest`).